routing.rst

```diff
@@ -2416,6 +2416,14 @@ with a locale. This can be done by defining a different prefix for each locale
             ;
         };
 
+.. tip::
+
+    If the special :ref:`_locale <routing-locale-parameter>` routing parameter
+    is set on any of the imported routes, that route will only be available
+    with the prefix for that locale. This is useful when you want to import
+    a collection of routes which contains a route that should only exist
+    in one of the locales.
```

Comment:
What about adding a code example instead? This could make the effect more clear (and we can still explain it with an inline comment)?

Reply:
I often use the "soccer" / "football" thing to illustrate the "default prefix expect one locale" 
* football (default)
* us : soccer

In the opposite situation, i guess you could use services in ecommerce that are often restricted to the country where the shop operates,  like "express-delivery",  cashback options, .. etc ?

Reply:
I have added an example with annotations and with attributes. I'm not sure if I should add examples for YAML, XML and PHP as well?